modules/bioinspired/src/transientareassegmentationmodule.cpp

```diff
@@ -166,7 +166,6 @@ class TransientAreasSegmentationModuleImpl : protected BasicRetinaFilter
 
     /**
      * access function
-     * @return the last segmentation result: a boolean picture which is resampled between 0 and 255 for a display purpose
```

Comment:
Just remove `@` specifier.

Internal code comments must not be a part of public API documentation.